.ci/azure/linux.yml

```diff
@@ -125,6 +125,7 @@ jobs:
     displayName: 'nGraph UT'
     continueOnError: false
 
+    # python3 $(WORK_DIR)/gtest-parallel/gtest_parallel.py $(BIN_DIR)/InferenceEngineUnitTests --workers=$(WORKERS_NUMBER) --dump_json_test_results=InferenceEngineUnitTests.json --gtest_filter=*smoke* -- --gtest_print_time=1
```

Comment:
why this change in removing mkl dnn functional tests?

Reply:
it is empty